packages/blockchain-wallet-v4-frontend/src/modals/Btc/SendBtc/FirstStep/template.success.js

```diff
@@ -235,90 +272,126 @@ const FirstStep = props => {
+                  <FormattedMessage
+                    id='modals.sendbtc.firststep.networkfee'
+                    defaultMessage='Network Fee'
+                  />
                   <span>&nbsp;</span>
-                  <PriorityFeeLink fee={priorityFeePerByte} />
-                </FeeOptionsContainer>
-              )}
-            </FeeFormLabel>
-            {feePerByteToggled && (
-              <FeePerByteContainer>
-                <Field
-                  name='feePerByte'
-                  component={NumberBoxDebounced}
-                  validate={[required, minimumOneSatoshi]}
-                  warn={[minimumFeePerByte, maximumFeePerByte]}
-                  errorBottom
-                  errorLeft
-                  unit='sat/byte'
-                  data-e2e='sendBtcCustomFeeInput'
+                  {!feePerByteToggled && (
+                    <Field
+                      name='feePerByte'
+                      component={SelectBox}
+                      elements={feePerByteElements}
+                    />
+                  )}
+                  {feePerByteToggled && (
+                    <FeeOptionsContainer>
+                      <RegularFeeLink fee={regularFeePerByte} />
+                      <span>&nbsp;</span>
+                      <PriorityFeeLink fee={priorityFeePerByte} />
+                    </FeeOptionsContainer>
+                  )}
+                </FeeFormLabel>
+                {feePerByteToggled && (
+                  <FeePerByteContainer>
+                    <Field
+                      name='feePerByte'
+                      component={NumberBoxDebounced}
+                      validate={[required, minimumOneSatoshi]}
+                      warn={[minimumFeePerByte, maximumFeePerByte]}
+                      errorBottom
+                      errorLeft
+                      unit='sat/byte'
+                      data-e2e='sendBtcCustomFeeInput'
+                    />
+                  </FeePerByteContainer>
+                )}
+              </FeeFormContainer>
+            </ColLeft>
+            <ColRight>
+              <ComboDisplay size='13px' coin='BTC'>
+                {totalFee}
+              </ComboDisplay>
+              <Link
+                size='12px'
+                weight={400}
+                capitalize
+                onClick={handleFeePerByteToggle}
+                data-e2e='sendBtcCustomFeeLink'
+              >
+                {feePerByteToggled ? (
+                  <FormattedMessage
+                    id='modals.sendbtc.firststep.cancel'
+                    defaultMessage='Cancel'
+                  />
+                ) : (
+                  <FormattedMessage
+                    id='modals.sendbtc.firststep.customizefee'
+                    defaultMessage='Customize Fee'
+                  />
+                )}
+              </Link>
+            </ColRight>
+          </FeeFormGroup>
+          {feePerByteToggled ? (
```

Comment:
`{feePerByteToggled && (` then get rid of line 375

Reply:
I didn't write that code, didnt want to touch something that was already there